lua/starfall/libs_cl/render.lua

```diff
@@ -472,6 +472,7 @@ function instance:cleanupRender()
 	render.ResetModelLighting(1, 1, 1)
 	render.DepthRange(0, 1)
 	render.SuppressEngineLighting(false)
+	render.SetWriteDepthToDestAlpha(true)
```

Comment:
Is the original state normally true?

Reply:
I'm going off of the example here https://wiki.facepunch.com/gmod/render.SetWriteDepthToDestAlpha which sets it to true first and then to false.